bpf/tests/ipsec_redirect_generic.h

```diff
@@ -27,11 +27,78 @@ int mock_ctx_redirect(const struct __sk_buff *ctx __maybe_unused,
 /* test constants */
 #define SOURCE_MAC mac_one
 #define DST_MAC mac_two
+#define SOURCE_NODE_IP v4_node_one
+#define SOURCE_NODE_IP_6 v6_node_one
 #define SOURCE_IP v4_pod_one
-#define SOURCE_IDENTITY 0xAB
+#define SOURCE_IP_6 v6_pod_one
+#define SOURCE_IDENTITY 0xAB0000
```

Comment:
(out-of-scope) I've always wondered how robust it is to pick such "random" numbers as identities. Or how easy it would be to have accidental conflicts with range-based checks like the CIDR-range check in `identity_is_cluster()`.